maplibre/src/lib.rs

```diff
@@ -1,3 +1,21 @@
+//! # Maplibre-rs
+//!
+//! A multi-platform library for rendering vector tile maps with WebGPU.
+//!
+//! Maplibre-rs is a map renderer that can run natively on MacOS, Linux, Windows, Android, iOS and the web.
+//! It takes advantage of Lyon to tessellate vector tiles and WebGPU to display them efficiently.
+//! Maplibre-rs also has an headless mode (*work in progress*) that can generate rasters.
+//!
+//! The official guide book can be found here #![doc = include_str!("description.md")]
```

Comment:
```suggestion
//! The official guide book can be found [here](https://maxammann.org/maplibre-rs/docs/)
```

Reply:
It doesn't seem to work with the include_str!. I think a simple link to the docs should work fine.